assets/css/layouts/_homepage.scss

```diff
@@ -4,4 +4,16 @@
   @media (min-width: 800px) {
     margin-top: calc(var(--l-gap) * 4);
   }
-}
\ No newline at end of file
+}
+
+#ways-to-help.spoiler {
+  .sub-headline {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+
+    .spoiler__toggle-btn {
+      color: #fff;
+    }
+  }
+}
```

Comment:
I wonder if we can make these style more reusable, and less tied to the specific instance?

Reply:
Do you mean both `.sub-headline` and `.spoiler__toggle-btn`?

Reply:
Yes, the combination of both. Aligned with the proposed update of the structure (https://github.com/selfdefined/web-app/pull/400#discussion_r621928910) you should be able to remove the nested styles altogether. Maybe add a utility class `.flex-space-between` to have it reusable and apply this to the header. Am I making sense? Have a bit of a brain freeze currently.

Reply:
Another thing, for line 16. We added a dark mode last year, and as such, we try to avoid hardcoded colour values. The [links in the box are styled in this file](https://github.com/selfdefined/web-app/blob/prod/assets/css/components/_box.scss#L17), maybe it works to apply the same styles to the button?

Reply:
As for utility classes, here are related commits: 641120c, ff605f6
As for wrong icon color, I choose accent color. I think it looks even cooler 😁 . 8b69cce